docs/import/add_disaster.js

```diff
@@ -370,112 +376,76 @@ function populateStateAssetPickers() {
 }
 
 /**
- * Initializes the select logic for poverty assets.
+ * Initializes the select interface for score assets.
  * @param {Array<string>} states array of state (abbreviations)
- * @return {Promise<boolean>} returns true after successful write to firestore.
  */
-function initializePovertySelectors(states) {
-  let assets = [];
-  for (const state of states) {
-    if (stateAssets.get(state)) {
-      assets = assets.concat(stateAssets.get(state));
-    }
-  }
+function initializeScoreSelectors(states) {
+  const headerRow = $('#score-asset-header-row');
+  const tableBody = $('#asset-selection-table-body');
+  tableBody.empty();
+  headerRow.empty();
 
-  if (assets.length) {
-    $('#score-asset-selection').show();
-  } else {
-    $('#score-asset-selection').hide();
+  // Initialize headers.
+  headerRow.append(createTd().html('Score Assets'));
+  for (const state of states) {
+    headerRow.append(createTd().html(state + ' Assets'));
```

Comment:
Does this mean that the assets for different states need to share the same column names? I'm assuming this is fine because I can't imagine you're getting data about one state from one source and from another state from another source. Just making sure.